requirements-dev.txt

```diff
@@ -2,3 +2,4 @@ git+https://github.com/asdf-format/asdf
 git+https://github.com/spacetelescope/roman_datamodels
 git+https://github.com/spacetelescope/rad
 astropy>=0.0.dev0
+numpy>=0.0.dev0
```

Comment:
This isn't quite right, see https://github.com/spacetelescope/romancal/pull/695#discussion_r1189076034 and https://github.com/spacetelescope/romancal/pull/695#discussion_r1189078751 for details.